zebra/zebra_vty.c

```diff
@@ -4446,6 +4466,9 @@ void zebra_vty_init(void)
 	install_element(CONFIG_NODE, &zebra_dplane_queue_limit_cmd);
 	install_element(CONFIG_NODE, &no_zebra_dplane_queue_limit_cmd);
 
+	install_element(CONFIG_NODE, &ip_table_range_cmd);
```

Comment:
I'm not sure what this means - doesn't the command only work in vrf submode?

Reply:
if vrf is default vrf, this command may also be useful. that is why it is also availble at config level ( for default vrf)